.travis.yml

```diff
@@ -82,6 +82,10 @@ before_install:
   - if [ "$REDIS_ENABLE_LZF" = true ] ; then printf "no\n" | pecl install lzf ; fi
   # Prepare system
   - if [ "$TEST_CONFIG" != "" ] ; then ./bin/.travis/prepare_unittest.sh ; fi
+    # Set up swap
+  - if [ "$BEHAT_OPTS" != "" ] ; then sudo /bin/dd if=/dev/zero of=/var/swap.1 bs=1M count=4096 ; fi
```

Comment:
feels like something we should do for all jobs for performance reasons

Reply:
Well, if jobs run without swap (default config for travis images iirc), you will be wasting time setting it up, making them marginally slower... In the travis jobs that I run f.e. for migration bundle, where I install ez as a dependency, I only activate swap for php 5.6, as otherwise composer craps out

Reply:
> Well, if jobs run without swap (default config for travis images iirc), you will be wasting time setting it up, making them marginally slower... In the travis jobs that I run f.e. for migration bundle, where I install ez as a dependency, I only activate swap for php 5.6, as otherwise composer craps out

If it's marginal, should it really matter?

Depends on the slowness I guess. If this operation takes more than one second, then indeed we can keep it as is. If it's less than a second of difference I would go for it globally. It would be beneficial overall even though not visible at first glance for most of the jobs.

Reply:
From what I saw during travis run, it was something between 10 to 20 seconds if I remember correctly, but that was single run, so dunno if it is valid then.

Reply:
> From what I saw during travis run, it was something between 10 to 20 seconds if I remember correctly, but that was single test, so dunno if it is valid then.

ok, then indeed let's switch as needed.

@mnocon one single request from my side: instead of adding triple times `- if [ "$BEHAT_OPTS" != "" ] ` please move it to one script, e.g. 
```
- if [ "$BEHAT_OPTS" != "" ] ; then ./bin/.travis/enable_swap.sh ; fi
```